donation-web/src/main/java/com/donation/web/DataController.java

```diff
@@ -53,22 +53,22 @@ public String populateDatabase() {
         Video video = new Video();
         video.setSponsor(sponsor);
         video.setTitle("IAM_1");
-        video.setLink(video.getTitle() + "__link");
+        video.setLink("C://Users/SQLI/Downloads/myvideo.mp4");
```

Comment:
please use avoid using absolute path... use relative path

Reply:
Done

Reply:
i tried using relative path i kept getting an error file not found. I tried adding the video to multiple places and kept getting the same error. I'm still trying to find a solution

Reply:
Ok I will merge the PR and keep this change as technical dept, please fix it separatly